modules/tinymce/src/core/main/ts/init/Init.ts

```diff
@@ -81,11 +81,11 @@ const initIcons = (editor: Editor) => {
   });
 };
 
-const initTheme = function (editor: Editor) {
-  const theme = editor.settings.theme;
+const initTheme = (editor: Editor) => {
+  const theme = Settings.getTheme(editor);
 
   if (Type.isString(theme)) {
-    editor.settings.theme = trimLegacyPrefix(theme);
+    editor.settings.theme = trimLegacyPrefix(theme); // Kept until a proper API can be made.
```

Comment:
Do we have a Jira for the "proper API" thing?

Reply:
I'm going to go out on a limb and say if you don't know, the answer is probably no.

Reply:
Added one.